app/models/provider_user_filter.rb

```diff
@@ -0,0 +1,3 @@
+class ProviderUserFilter < ApplicationRecord
```

Comment:
Probably a bit late for this input but how about 
```suggestion
class PageFilter < ApplicationRecord
```
Could we see a world where Support or Candidate users may use this?